src/roles/common/templates/vnsutil.j2

```diff
@@ -58,6 +58,50 @@ vsd_fqdn: {{ common.vsd_fqdn_global }}
 image_path: {{ image_path | default("MISSING") }}
 image_file_name: {{ image_file_name | default("MISSING") }}
 
+{% if item.target_server_type is match("openstack") %}
+
+{% if item.openstack_mgmt_network is defined %}
+openstack_mgmt_network: {{ item.openstack_mgmt_network }}
+{% endif %}
+
+{% if item.openstack_mgmt_subnet is defined %}
+openstack_mgmt_subnet: {{ item.openstack_mgmt_subnet }}
+{% endif %}
+
+{% if item.openstack_data_network %}
+openstack_data_network: {{ item.openstack_data_network }}
+{% endif %}
+
+{% if item.openstack_data_network %}
```

Comment:
Did you mean for this to be a duplicate of line 71?

Reply:
No, it's fixed now